src/main/java/com/aquarium/aquarium_backend/databaseTables/DictTableValues/EnvironmentalRequirements.json

```diff
@@ -1,10 +1,20 @@
 [
   {
     "name": "PH",
-    "unit": ""
+    "unit": "",
+    "value": 4,
+    "fish": [
+      "Guppy",
+      "Molly"
+    ]
   },
   {
     "name": "Temperature",
-    "unit": "°C"
+    "unit": "°C",
+    "value": 25,
+    "fish": [
+      "Guppy",
+      "Molly"
+    ]
```

Comment:
I don't know about possibilities to nest json but I don't think it's a good idea to create a lot of Temperature requirements instead of nested values for const unit and name.

Reply:
I think we need to rethink the way fish type and requirements tables connect because currently its really messy and adding list of names is just temporary solution (new issu3 should be created for that)